lib/shortcodes/list/class-list-shortcode.php

```diff
@@ -105,10 +111,18 @@ public function get_rendered_shortcode( $atts, $content ) {
 
 			if ( ! empty( $post_items ) ) {
 
+				if ( ! empty( $atts['title'] ) ) {
+
+					$html .= '<' . $atts['title_tag'] . '>' . $atts['title'] . '</' . $atts['title_tag'] . '>';
```

Comment:
It would be ideal to escape both `$atts['title_tag']` and `$atts['title']` here to prevent accidental HTML breakage.